shared_model/utils/string_builder.hpp

```diff
@@ -73,6 +72,15 @@ namespace shared_model {
       static const std::string singleFieldsSeparator;
       static const std::string initSeparator;
       static const std::string spaceSeparator;
+
+      template<typename T>
+      inline void appendPartial_(T const& value) {
```

Comment:
https://google.github.io/styleguide/cppguide.html#Naming
Please remove trailing underscore for consistency with the codebase.